Network/Packet.h

```diff
@@ -233,7 +231,6 @@ namespace OP2Internal
 			GameMessage gameMessage;
 		};
 
-	public:
 		// Member functions
 		int Checksum();

```

Comment:
Can this be marked as const since I'm guessing it just returns the checksum?

-Brett

Reply:
That's a very good point.